config/forms/2021_2022.json

```diff
@@ -1121,14 +1144,20 @@
                   },
                   "conditional_for": {
                     "property_postcode": ["Yes"]
-                  }
+                  },
+                  "hidden_in_check_answers": true
                 },
                 "property_postcode": {
                   "check_answer_label": "Postcode",
                   "header": "",
                   "hint_text": "",
                   "type": "text",
-                  "width": 5
+                  "width": 5,
+                  "inferred_answers": { "la": { "is_la_inferred": true } },
+                  "inferred_check_answers_value": {
+                    "condition": { "postcode_known": "No" },
+                    "value": "Not known"
+                  }
```

Comment:
This is the only actual change, the rest is formatting..

Reply:
Could we add this to the documentation? Possibly also to the JsonSchema so it validates (not sure if anyone is using that/finding it useful though?)